content/index.md

```diff
@@ -6,14 +6,18 @@ type: index
 
 Welcome! Cloud Foundry is an open source Platform-as-a-Service (PaaS) system for managing the deployment of apps, services, and background tasks. 18F is starting to use it for many of our development and production systems.
 
-## Help!
+## Sections
 
-Find the section you need on the left. Reach out in [#cloud-gov-support](https://18f.slack.com/messages/cloud-gov-support/) or [open an issue](https://github.com/18F/cloud-foundry-notes/issues/new) if you need anything. If you're not part of 18F but want to talk with us about fun DevOps stuff, [join our #devops-public Slack channel](https://chat.18f.gov/).
+(In the sidebar on the left.)
 
-## Public domain
+### Introduction
 
-This project is in the worldwide [public domain](https://github.com/18F/cloud-foundry-notes/blob/master/LICENSE.md). As stated in [CONTRIBUTING](https://github.com/18F/cloud-foundry-notes/blob/master/CONTRIBUTING.md):
+Learn more about cloud.gov.
 
-> This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
->
-> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
+### Usage
+
+Learn how to deploy applications to cloud.gov.
+
+### Contributing
+
+Intended for cloud.gov operations staff, but made public for others to learn how we run the platform.
```

Comment:
Don't love my various descriptions here...curious if others can think of better wording, or if they are helpful at all.